src/registrar/tests/common.py

```diff
@@ -0,0 +1,29 @@
+from django.conf import settings
+from django.contrib.auth import get_user_model, login
+
+
+class MockUserLogin:
+    def __init__(self, get_response):
+        self.get_response = get_response
+
+    def __call__(self, request):
+        if request.user.is_anonymous:
+            user = None
+            UserModel = get_user_model()
+            username = "Testy"
+            args = {
+                UserModel.USERNAME_FIELD: username,
+            }
+            try:
+                user = UserModel.objects.get(**args)
```

Comment:
**suggestion (non-blocking):** I think that this `try/except` block could idiomatically be replaced with `UserModel.objects.get_or_create()`?

Reply:
Good point. I've changed it.